uber/config.py

```diff
@@ -136,6 +136,19 @@ def SUPPORTER_COUNT(self):
                                                 sa.Attendee.amount_paid >= self.SUPPORTER_LEVEL).count()
             return individual_supporters + group_supporters
 
```

Comment:
fixed, though I had to add something, see below.  @EliAndrewC let me know what you think